### ISSUE TITLE
Group languages in a drop down menu (5 pts)

### ISSUE THREAD

**Issue by txau** [huridocs/uwazi#1899]
Some instances are using so many languages our current solution is not good enough.

![screenshot from 2018-09-15 09-40-29](https://user-images.githubusercontent.com/489831/45583961-7a07bf00-b8cb-11e8-9453-0a803c949d92.png)

Making sure it is responsive.

**Comment by txau** [comment]
- On mobile put the languages into the drop down menu.
- Make it vertical.
- Display language name.
- On desktop is a separate drop-down menu replacing the current one.

**Comment by simonfossom** [comment]
Design for this is here: https://www.figma.com/file/xjsz7r65epN4B0b96zilac/UWAZI-redesigns?node-id=133%3A101

**Comment by elric-wamugu** [comment]
> Design for this is here: https://www.figma.com/file/xjsz7r65epN4B0b96zilac/UWAZI-redesigns?node-id=133%3A101

@simonfossom getting a file not found error on the figma link

**Comment by simonfossom** [comment]
@elric-wamugu please try again

**Comment by RafaPolit** [comment]
@simonfossom The new design of grouped langauges in mobile view or screen view prevents the user from knowing what languages are available.  How are we solving that from the point of view of accessibility?

**Comment by simonfossom** [comment]
@RafaPolit One of the problems we are trying to solve for here is that there might be dozens of languages, so we have to group languages into a submenu and so we did visually. 'Language' icon is also there to help.

When you say accessibility, I presume you're addressing the screen-reader issue. I've talked to @elric-wamugu about it. There is an ARIA tag that can declare which language an element is in. This allows us to check what is default browser language and render "Choose language" text element in the default browser language so that the screen-reader can read it and a user can hear it in their language. (Ofc, if browser language doesn't match any of the instance languages, then this text should be read in the instance's default language.

The same is for each language of the language menu: 'English' is read in English, 'Français' in French, etc.

Hope this helps.
